dfaker.py

```diff
@@ -486,24 +112,26 @@ def settings(start_time, params):
 start_time = datetime(d.year, d.month, d.day, 
                     d.hour, d.minute, tzinfo=timezone(params['zone']))
 
-cbg_gluc, cbg_time, smbg_gluc, smbg_time = apply_loess(params, solution)
-cbg_timesteps = make_timesteps(start_time, cbg_time)
-smbg_timesteps = make_timesteps(start_time, smbg_time)
+cbg_gluc, cbg_time, smbg_gluc, smbg_time = cbg.apply_loess(params, solution)
+cbg_timesteps = tools.make_timesteps(start_time, cbg_time)
+smbg_timesteps = tools.make_timesteps(start_time, smbg_time)
+
+b_carbs, b_carb_timesteps, w_carbs, w_carb_timesteps, w_gluc = bolus.generate_boluses(solution, start_time)
 
-b_carbs, b_carb_timesteps, w_carbs, w_carb_timesteps, w_gluc = generate_boluses(solution, start_time)
+#make settings 
+settings_data = settings.settings(start_time, params)
+#make basal values
+basal_data = basal.scheduled_basal(start_time, params)
+#make bolus values 
+bolus_data = bolus.bolus(start_time, b_carbs, b_carb_timesteps, params)
+#make wizard events
+wizard_data = wizard.wizard(start_time, w_gluc, w_carbs, w_carb_timesteps, params)
+#make cbg values 
+cbg_data = cbg.cbg(cbg_gluc, cbg_timesteps, params)
+#make smbg values 
+smbg_data = smbg.smbg(smbg_gluc, smbg_timesteps, params)
```

Comment:
Another thing that might make the calls from `dfaker` to the dependent type modules a tad nicer & cleaner would be importing just the function you're using, so instead of:
```
import bolus
...
bolus_data = bolus.bolus(start_time, b_carbs, b_carb_timesteps, params)
```

You could have:
```
from bolus import bolus
...
bolus_data = bolus(start_time, b_carbs, b_carb_timesteps, params)
```